src/Wrapper/LambdaWrapper.js

```diff
@@ -25,10 +25,32 @@ import ResponseModel from '../Model/Response.model';
 export const handleError = (di, error) => {
   const logger = di.get(DEFINITIONS.LOGGER);
 
+  let errorBody = error;
+
+  // While handling an error, lambda wrapper should
+  // recognise axios errors and trim down the information
+  if (errorBody.isAxiosError) {
+    try {
+      // only keep error.config, error.response.status, error.response.data
+      errorBody = {
+        config: error.config,
+        response: {
+          status: error.response.status,
+          data: error.response.data,
+        },
+      };
+    } catch {
+      // if the axios error is incomplete
+      // for any reason, we don't want
+      // error handling to fail
+      errorBody = error;
```

Comment:
It's pretty common for axios errors to no have`.response` e.g. when there's a network error or timeout. These errors will have `.request` but not `.response`. So would probs be good to handle those cases without just using the full error.

Also, would be good to keep `error.message` too maybe?

Reply:
Oh and maybe some of the `request` info can be kept, might be useful in development? E.g. url, payload, query parameters etc

Reply:
> Oh and maybe some of the request info can be kept, might be useful in development? E.g. url, payload, headers, query parameters etc

They come with `error.config`

Reply:
> It's pretty common for axios errors to not have`.response` e.g. when there's a network error or timeout. These errors will have `.request` but not `.response`. So would probs be good to handle those cases without just using the full error.
> 
> Also, would be good to keep `error.message` too maybe?

Good points. On it!

Reply:
Nice one. Good to know re `.config`

Reply:
b08c3228fbee3a8f2ff81b08fcdd1413fe8b174f